doc/exts/pylint_messages.py

```diff
@@ -0,0 +1,207 @@
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+# For details: https://github.com/PyCQA/pylint/blob/main/LICENSE
+
+"""Script used to generate the messages files."""
+
+import os
+from collections import defaultdict
+from typing import DefaultDict, Dict, List, NamedTuple, Optional, Tuple
+
+from sphinx.application import Sphinx
+
+from pylint.checkers import initialize as initialize_checkers
+from pylint.extensions import initialize as initialize_extensions
+from pylint.lint import PyLinter
+from pylint.message import MessageDefinition
+from pylint.utils import get_rst_title
+
+# PACKAGE/docs/exts/pylint_extensions.py --> PACKAGE/
+PYLINT_BASE_PATH = os.path.dirname(
+    os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+)
+"""Base path to the project folder"""
+
+
+MSG_TYPES = {
```

Comment:
``MSG_TYPES`` can be imported from ``pylint.constants``

Reply:
`MSG_TYPES` in `pylint.constants` uses `info` instead of `information`. I checked and we tend to use `information-level` in our changelogs and docs so I thought it would be best to do so here as well. That's why I created a new constant.

Reply:
Maybe we could create the new constant from the old one (or changing the value to information in the old one is reasonable) ?

Reply:
I think its part of the public API? 😅 Changing something called a constant might warrant a deprecation period haha

Reply:
```suggestion
MSG_TYPES_DOC = {k:v if v != "info" else "information" for k, v in MSG_TYPES.items()}
```